### ISSUE TITLE
Prune babel cache?

### ISSUE THREAD

**Issue by gdborton** [babel/babel-loader#634]
## **BEFORE YOU SUBMIT** please read the following:

**I'm submitting a feature request**
Would it be possible to add a way to prune the babel-loader cache after webpack runs?  We've started uploading/downloading our cache in CI to help performance across builds, but the babel-cache grows unbounded and eventually bogs down our CI.

**Babel Loader Version**:
7.1.4

**Current behavior:**
babel cache grows unbounded

**Expected/desired behavior:**
It doesn't.

**Comment by loganfsmyth** [comment]
We could consider having some kind of LRU prune step the first the time loader runs in a given process, or something. It's a little ugly no matter what unfortunately.

**Comment by gdborton** [comment]
Agreed about the ugliness, especially since we can have more than one compiler/process trigger babel-loader.
